source/ShipInfoDisplay.cpp

```diff
@@ -350,6 +354,23 @@ void ShipInfoDisplay::UpdateAttributes(const Ship &ship, const Depreciation &dep
 		* (1. + attributes.Get("hull heat multiplier")) : 0.;
 	heatTable.push_back(Format::Number(60. * (shieldHeat + hullHeat)));
 
+    // Add up the maximum possible energy and heat changes; add the total to the table.
+    attributesHeight += 20;
+	const double overallEnergy = idleEnergyPerFrame
```

Comment:
why do ident here?

Reply:
I don't know... It's not showing up for me in Xcode.

Reply:
Problem was I had spaces instead of a tab to indent "attributesHeight." Should be fixed now.